src/include/cache.h

```diff
@@ -13,7 +13,7 @@
 #define	WT_EVICT_INT_SKEW  (1<<20)	/* Prefer leaf pages over internal
 					   pages by this many increments of the
 					   read generation. */
-#define	WT_EVICT_WALK_PER_FILE	 10	/* Pages to queue per file */
+#define	WT_EVICT_WALK_PER_FILE	 50	/* Pages to queue per file */
```

Comment:
Is this change still necessary?  How much of the speedup in the original workload is due to this change?

The reason I ask is that there are workloads where eviction spends too much time in files that don't have many candidate pages.  That was the motivation for keeping this value small.